package.json

```diff
@@ -29,8 +31,11 @@
     "@wordpress/eslint-plugin": "^11.0.0",
     "@wordpress/prettier-config": "^1.1.1",
     "chalk": "^4.1.2",
+    "eslint": "^8.12.0",
     "glob": "^7.2.0",
+    "husky": "^7.0.0",
     "jest": "^27.3.1",
+    "lint-staged": "^12.3.7",
```

Comment:
`lint-staged` would be optional depending on if we just want to call `pnpm nx affected:lint` and leave it up to that command.
Otherwise we might want to do something like:
```
"*.php": [
      "php -d display_errors=1 -l",
      "pnpm nx affected:lint-php"
    ],
    "*.{js,ts,tsx}": [
      "pnpm nx affected:lint-js"
    ]
```